### ISSUE TITLE
.NET Core support (and Linux and macOS)

### ISSUE THREAD

**Issue by promontis** [moethu/SketchUpNET#35]
AFAIK SketchUpNET only works on Windows right? You are using a vcxproj project, that has C++ sources. 

This articles discusses how to port a C++ project to .NET Core: https://docs.microsoft.com/en-us/dotnet/core/porting/cpp-cli. However, it notes that C++/CLI support for .NET Core is Windows only. Support for macOS or Linux isn't probably to come (for a long time).

They recommend using platform invoke to support .NET core. What's your view on this? Do you have any plans on making it work for .NET Core (and thereby Linux and macOS)?

**Comment by moethu** [comment]
Good idea, I'll give it a try but I doubt that the SkechUp C++ API (slapi) will work on dotnetcore.

**Comment by moethu** [comment]
Just checked this one again, currently there is no planned support of C++/CLI for linux mac in any near future. One option would be to get rid of C++/CLI and use Pinvoke instead. Would be a larger rewrite but could be worth it. I'd first try a simple implementation, eg. reading one vertex to see if the SketchUp C API works in Linux.
